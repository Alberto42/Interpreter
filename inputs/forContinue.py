i = 0
x = 0
for x in range(0,9) {
    y = x
    if x * 2 > 5 {
        continue
    }
    i = i + 1
}