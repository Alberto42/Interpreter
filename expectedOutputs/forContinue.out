Right (OK,fromList [(Ident "i",VInt 3),(Ident "x",VInt 0),(Ident "y",VInt 9)])
